test/common/router/config_impl_test.cc

```diff
@@ -166,7 +166,9 @@ class ConfigImplTestBase {
     return factory_context_.scope().symbolTable().toString(name);
   }
 
-  std::string responseHeadersConfig(const bool most_specific_wins, const bool append) {
+  std::string responseHeadersConfig(
```

Comment:
I think you should have some DEPRECATED_FEATURE_TEST in here somewhere to make sure you test the old behavior.

Reply:
Added, Thanks!